src/exchanges/fetch.ts

```diff
@@ -99,6 +97,13 @@ const createFetchSource = (operation: Operation) => {
         abortController !== undefined ? abortController.signal : undefined,
     };
 
+    if (fetchOptions.method === 'GET' && operation.operationName === 'query') {
```

Comment:
This is kind off implicit logic so I'll probably move this to `client.useGetForQueries` instead and attach it to the `Operation` this will grant the benefit of us also being able to clearly document this

Reply:
I think the naming here as a little shaky, but I’d expect that we’re going a little off-spec and standards here we could also make some assumptions and call the option `preferGetMethod`